packages/DropdownMenu/stories/examples/DropdownMenuExample.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { select } from "@storybook/addon-knobs";
+import L10n from "@paprika/l10n";
+import DropdownMenu from "../../src";
+
+const DropdownMenuExample = () => {
+  const handleConfirm = handleCloseMenu => {
+    handleCloseMenu();
+  };
+
+  const handleCancel = handleCloseMenu => {
+    handleCloseMenu();
+  };
+
+  return (
+    <L10n locale={select("locale", ["en", "de", "es", "fr", "ja", "pt", "zh"], "en")}>
+      <DropdownMenu
+        align="bottom"
+        renderTrigger={({ isOpen, handleOpenMenu }) => (
+          <DropdownMenu.Trigger isOpen={isOpen} handleOpenMenu={handleOpenMenu}>
+            Trigger
+          </DropdownMenu.Trigger>
+        )}
+      >
+        <DropdownMenu.Item onClick={() => {}}>Edit</DropdownMenu.Item>
+        <DropdownMenu.Item onClick={() => {}}>Duplicate</DropdownMenu.Item>
+        <DropdownMenu.Item isDestructive isDisabled onClick={() => {}}>
+          Google
```

Comment:
If this is for Google, it should be a "link", right?  
Also, right now the styling is not "destructive" or "disabled", it just looks like a regular item (but isn't clickable).  I'd recommend hooking up the `onClick` props to a Storybook `action`, so you can verify if they are actually firing or not, as intended.  Take a look at the Button's story for an example:
https://github.com/acl-services/paprika/blob/master/packages/Button/stories/examples/Basic.js#L19

Reply:
Also, can we change Goggle for Galvanize :P ...

Reply:
...and here's a URL you can use to test it:
https://youtu.be/IdkCEioCp24?t=73